certs/variables.tf

```diff
@@ -39,9 +45,10 @@ variable "ami_region_lookup" {
   type = "map"
 
   default = {
-    us-east-1      = "ami-d66995bb"
-    ap-northeast-1 = "ami-4803ec29"
-    us-west-2      = "ami-4308a323"
+    us-east-1      = "ami-6934c804"
```

Comment:
This rolls back to the older openvpn AMIs, but public. The `custom_ami` can override these temporarily and doesn't cause any noise in the stack plan.